### PR TITLE
[W5.7][W10-2] Solayappan Meenakshi

### PR DESCRIPTION
**Pull request by Meena567**

### CODE REVIEW

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -47,9 +43,19 @@ public CommandResult execute() {
...
+        keywords = Set.of(keywordsArr);
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            String[] wordsInNameArr = wordsInName.toArray(new String[0]);
+            for(int i = 0; i < wordsInNameArr.length; i++) {
+                wordsInNameArr[i] = wordsInNameArr[i].toLowerCase();
+            }
+            wordsInName = Set.of(wordsInNameArr);
```
Comment (navekom) [inline src/seedu/addressbook/commands/FindCommand.java:58]: Why not just put a List<String> of lowercase names into the new HashSet<>( ... )?
That way, it reduces the trouble of you converting it to array and then back to set again.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -47,9 +43,19 @@ public CommandResult execute() {
      * @return list of persons found
```
Comment (navekom) [inline src/seedu/addressbook/commands/FindCommand.java:43]: Gotta update your javadocs too!

**Review by navekom — COMMENTED** [review]
Interesting feature. Took me a while to understand the modification tho.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 3]
```diff
@@ -1,10 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (damithc) [inline src/seedu/addressbook/commands/FindCommand.java:3]: [Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation:exclamation:

**Review by damithc — APPROVED** [review]
:+1: Good effort on this PR @Meena567. In particular, good job on,
* updating docs
* updating I/O tests
* updating JUnit tests
* keeping the PR relatively clean

As before, please close the PR after viewing comments.

**Pull request closed without merging**